### ISSUE TITLE
screenOrientation portrait gets removed

### ISSUE THREAD

**Issue by LianVyv** [maitrungduc1410/react-native-video-trim#67]
Hi @maitrungduc1410 

I have set my app to be only portrait on Android

```Java
<activity
    android:name=".MainActivity"
    android:label="@string/app_name"
    android:screenOrientation="portrait"
    android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
    android:windowSoftInputMode="adjustResize">
    ...
</activity>
```

But I have notice when I open trim then it removes my portrait screenOrientation and it will switch my app to landscape when I turn the phone. 

- Library version: 2.0.0 & 2.2.0
- Device details: Infinix HOT 30 PLAY running on Android XOS 12.6.0
- minSDKVersion: 24
- RN version: 0.74.1

Steps I took to reproduce.

1. Open my app
2. Turn to landscape app stays in portrait
3. Open camera and turn to landscape app stays in portrait
4. Record video and open `showEditor(data.uri)`
5. On Trim screen turn to landscape stays in portrait
6. Click cancel then camera screen shows, turn the phone to landscape then app changes to landscape
7. Go back to main screen turn phone to landscape then app turns to landscape. 

Is there a setting that I should be adding to avoid this?

**Comment by maitrungduc1410** [comment]
@LianVyv support landscape is very tricky, that's why currently only portrait is supported

**Comment by maitrungduc1410** [comment]
I have plan to work on this but not anytime soon

**Comment by LianVyv** [comment]
@maitrungduc1410 I think you miss understood. I don't want landscape I only want potrait but the library remove my portrait only options and then it goes into landscape. I don't want to use landscape at all.

**Comment by LianVyv** [comment]
@maitrungduc1410 

Here is a video: https://drive.google.com/file/d/17jLLkze_Ai7Jqe7ogj9qA4e5axNCRpFP/view?usp=sharing

You will see before I open the trimming it won't go into landscape after I open it and click cancel then it goes into landscape.

**Comment by maitrungduc1410** [comment]
I see,

That view is default behaviour from the platform (Android)

Can give me your input video? Just want to have a look (but again, it's default behaviour of the platform)

**Comment by LianVyv** [comment]
@maitrungduc1410 what do you mean with it is default behaviour

if I set my android to be portrait then it should never change `android:screenOrientation="portrait"` so I'm not sure how this can be default behaviour your library should not change my apps screenOrientation right?

As you can on my video I'm recording a video then inputing it into the trim function so it is not a specific video.

**Comment by maitrungduc1410** [comment]
What I mean is

On Android, Im using VideoView, and simply pass the video url. That means what is displayed is what the platform gives

**Comment by maitrungduc1410** [comment]
My library doesn't manipulate these things, at all, I only focus on trimming (getting start/end time then trim)

**Comment by LianVyv** [comment]
Mmm okay I get you. Then something else is doing this. 

For anyone else having this issue I have hacked it for now by locking by app to portrait when I'm done with the trimming of the videos. 

I used react-native-orientation-locker and then called `Orientation.lockToPortrait();`

I will jump into the code later when I'm done with my current tasks to see if I can find what is causing this issue on Android.
